<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenPrincipale</class>
 <widget class="QWidget" name="FenPrincipale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>661</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FenPrincipale</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;
color:black;</string>
  </property>
  <widget class="QFrame" name="gridFrame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1221</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>25</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>25</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>710</width>
          <height>620</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>685</width>
          <height>620</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableWidget {
	background-color:#D8D7D7;
	border-style:none;
	border-radius:40px;
	color:black;
}

QTableWidget::item:selected{
	border-style:none;
	background-color: rgba(0, 0, 0, 0);
}

  QScrollBar::handle:vertical {
        background-color:rgba(127, 127, 127, 0);
    }

QScrollBar{
	width:0px;
	background-color:rgba(193, 193, 193, 0);
}

QScrollBar::handle{
	border-radius:5px;
}

    QScrollBar::add-line:vertical {
        height: 0px;
        subcontrol-position: bottom;
        subcontrol-origin: margin;
    }
    QScrollBar::sub-line:vertical {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop: 0  rgb(32, 47, 130), stop: 0.5 rgb(32, 47, 130),  stop:1 rgb(32, 47, 130));
        height: 0 px;
        subcontrol-position: top;
        subcontrol-origin: margin;
    }</string>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::NoPen</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vertiL">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>100</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="bottomMargin">
             <number>100</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="topMargin">
               <number>25</number>
              </property>
              <property name="bottomMargin">
               <number>25</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Veuillez scanner votre carte d'employ√©</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txfEmploye">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit{
	border: 2px solid rgb(112, 112, 112);
	border-radius: 15px;
	background-color: white;
	padding-left:20px;
}</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="topMargin">
               <number>35</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>49</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Veuillez scanner le code barre de l'objet</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txfObjet">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit{
	border: 2px solid rgb(112, 112, 112);
	border-radius: 15px;
	background-color: white;
	padding-left:20px;
}</string>
                </property>
                <property name="inputMask">
                 <string/>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>15</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="topMargin">
               <number>40</number>
              </property>
              <item>
               <widget class="QPushButton" name="btnAffect">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                  <horstretch>30</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI Emoji</family>
                  <pointsize>17</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
border-radius:22px;
background-color:rgb(29, 67, 219);
color:white;
}

QPushButton:hover{
background-color:rgb(22, 50, 162);
}</string>
                </property>
                <property name="text">
                 <string>Affecter</string>
                </property>
                <property name="shortcut">
                 <string>Enter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDeAffect">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                  <horstretch>30</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>17</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
border-radius:22px;
background-color:rgb(29, 67, 219);
color:white;
}

QPushButton:hover{
background-color:rgb(22, 50, 162);
}</string>
                </property>
                <property name="text">
                 <string>D√©saffecter</string>
                </property>
                <property name="shortcut">
                 <string>Ctrl+Return</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnQuitter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-radius:22px;
background-color: rgb(209, 122, 90);
color:white;
}

QPushButton:hover{
background-color:rgb(176, 103, 76);
}
</string>
            </property>
            <property name="text">
             <string>Quitter</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+Q</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
